### ISSUE TITLE
Unexpected configuration generated when setting size factor scaling >1

### ISSUE THREAD

**Issue by calum-green** [projectdissolve/dissolve#370]
**Describe the bug**
When generating a simple random mix configuration of an optimised molecule such as benzene, the generated configuration is absolutely fine. However, if the size factor scaling->requested is set to above 1 such as 10 and a restart file is saved, when the restart file is loaded with the input file, the configuration of the system completely changes and bugs out with bonds all over the place. This means an equilibration sim cannot be ran using the much faster command line version as the size factor scaling must manually be changed in the gui and ran from there to avoid this issue

**To Reproduce**
Steps to reproduce the behaviour (if necessary):
1. Create simple random mix of molecule
2. Change size factor scaling->requested to 10
3. Save restart file (only way to get size factor scaling to not revert to 1) and input file
4. Quit dissolve and launch the input and restart file in either gui or command line version
5. Observe the configuration of the system has bugged dramatically and size factor scaling-> requested is 10

**Expected behaviour**
The configuration of the system should not change when loading it in with size factor scaling =/= 1

**Screenshots**
Screenshots attached of input file and configuration saved, and what the configuration changes to when it is loaded in

![Benzene323Bug1](https://user-images.githubusercontent.com/70141032/92711740-b4888300-f350-11ea-8f47-4bad1fd15915.png)
![Benzene323Bug2](https://user-images.githubusercontent.com/70141032/92711749-b6eadd00-f350-11ea-8be0-5eb893747309.png)

**Operating System:**
 - OS:  Windows and Linux command line
 - OS version / flavour: Windows 10

**Comment by rprospero** [comment]
I'm having a bit of trouble reproducing this bug.  Here's the steps that I've followed:

1. Open the `dissolve-gui` with no command line argument
2. In the menu, File->Open Local
3. open examples/water/input.txt
4. Click on the `Bulk` tab
5. Set the `Size Factor Scaling Requested` to 10.7
6. In the menu, Simulation->Step Five
7. Observe that the scale has changed in the 3d visualisation
8. In the menu, Simulation->Save Restart Point
9. Save as debug.txt
10. In the menu, File->Quit
11. Start dissolve gui with `build/bin/dissolve-gui examples/water/input.txt -t debug.txt`
12. Click on the `Bulk` tab
13. Confirm that the visualisation appears pretty much the same as when we started.

**Comment by calum-green** [comment]
I think the issue lies between steps 5 and 6 in your method. If you set the scaling factor to 10.7 and then save the restart file, when you reload the input file and the restart file the bug is observed. If you let the sim run for any number of steps with the scaling factor set before saving then when you save the restart and reload it the bug is not observed - this is essentially how I have been working around it.

I have encountered the issue most when I am generating the molecular ensemble in the dissolve-gui on my own laptop before sending it the input and restart file over to a machine I ssh into and let it equilibrate on. As far as I know, the restart file must be saved to maintain the value of the size factor when sending between the two machines.

I tried with the water example as you did and also the benzene data I have been working on and observed the bug on both systems.
